config.go

```diff
@@ -6,8 +6,11 @@ import (
 	"github.com/TankerHQ/identity-go/v3/internal/app"
 )
 
+// Config wraps information about an app
 type Config struct {
+	// AppID is the ID of the app corresponding to this config
 	AppID     string
+	// AppSecret is the app private signature key used to sign identities
```

Comment:
Not sure we should go into such details. From the point of view of the user, the app secret is opaque, they get it from the dashboard and that's all.

Reply:
They can get these details by reading the spec, but we don't want anyone not prepared to do cryptography themselves. We don't expose cryptographic details in the API and we don't want someone to think "oh, this is a key, let me use that somewhere else in my app" and possibly write something very insecure.
You can just say something like "AppSecret is a secret used to create identities"

Reply:
Okay thanks for the details! I'll update that :D